provider/aws/service_test.go

```diff
@@ -28,9 +28,47 @@ func TestServiceWebhookURL(t *testing.T) {
 	}
 }
 
+func TestServiceList(t *testing.T) {
```

Comment:
This test seems badly named and in the wrong file. Did the rest of these tests get renamed?